src/opera/commands/init.py

```diff
@@ -0,0 +1,107 @@
+import argparse
+from os import path
+from pathlib import Path, PurePath
+from zipfile import ZipFile, is_zipfile
+
+import yaml
+
+from opera.error import DataError, ParseError
+from opera.parser import tosca
+from opera.storage import Storage
+from opera.parser.tosca.csar import CloudServiceArchive
+
+
+def add_parser(subparsers):
+    parser = subparsers.add_parser(
+        "init",
+        help="Initialize the deployment environment for the service template or CSAR"
+    )
+    parser.add_argument(
+        "--instance-path", "-p",
+        help=".opera storage folder location"
+    )
+    parser.add_argument(
+        "--inputs", "-i", type=argparse.FileType("r"),
+        help="YAML file with inputs",
+    )
+    parser.add_argument("csar",
+                        type=argparse.FileType("r"),
+                        help="Cloud service archive or service template file")
+    parser.set_defaults(func=init)
+
+
+def init(args):
+    if args.instance_path and not path.isdir(args.instance_path):
+        raise argparse.ArgumentTypeError("Directory {0} is not a valid path!".format(args.instance_path))
+
+    storage = Storage(Path(args.instance_path) / ".opera") if args.instance_path else Storage(Path(".opera"))
```

Comment:
```suggestion
    storage = Storage(Path(args.instance_path)) if args.instance_path else Storage(Path(".opera"))
```
As discussed in the main thread.

Reply:
Resolved.